### PR TITLE
Refactoring of modal, unit testing, linting

### PR DESCRIPTION
**Pull request by rodrigo-barraza**
*Issue #:*
https://github.com/bcgov/entity/issues/<Put the github issue number here>

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/manage-business/manage-business-dialog/ManageBusinessDialog.vue [hunk 1]
```diff
@@ -436,6 +465,11 @@ export default defineComponent({
       return isBusinessLegalTypeFirm.value && props.isStaffOrSbcStaff
     })
 
+    const isAuthorizationRequestSentDialogOpen = computed(() => {
+      console.log('fire')
```
Comment (seeker25) [inline auth-web/src/components/auth/manage-business/manage-business-dialog/ManageBusinessDialog.vue:469]: console.log

File: auth-web/tests/unit/components/ManageBusinessDialog.spec.ts [hunk 2]
```diff
@@ -243,4 +239,53 @@ describe('ManageBusinessDialog Component', () => {
...
+        addBusiness: vi.fn(),
+        updateBusinessName: vi.fn(),
+        updateFolioNumber: vi.fn()
+      }
+    }
+
+    const store = new Vuex.Store({
+      strict: false,
+      modules: {
+        org: orgModule,
```
Comment (seeker25) [inline auth-web/tests/unit/components/ManageBusinessDialog.spec.ts:268]: This isn't right, the component uses Pinia not Vuex

**Comment by seeker25** [comment]
Missing the UX change for Ethan

**Comment by Jxio** [comment]
missing input length validation:
image
image

**Comment by seeker25** [comment]
@rodrigo-barraza could you restart this on `main` as well, we just merged the feature branch

**Comment by seeker25** [comment]
Help dialog needs some TLC:

image

image


Also noticed a weird bug with selecting name request, hitting cancel:

image

Switching back to Business:

image

**Comment by seeker25** [comment]
We also have no loading screen:

image



Also this URL doesn't populate the orgId in the prop correctly:

`https://dev.account.bcregistry.gov.bc.ca/account/business/business?accountid=2662`

but this one does:

`https://dev.account.bcregistry.gov.bc.ca/account/2662/business`

**Pull request closed without merging**